qt/src/OutputTextEdit.cc

```diff
@@ -217,7 +217,7 @@ void OutputTextEdit::paintEvent(QPaintEvent* e) {
 			rect.setLeft(startLine.cursorToX(startLinePos) - 0.5);
 			painter.drawRect(rect);
 
-			// Draw selections inbetween
+			// Draw selections between
```

Comment:
This one was actually correct.

Reply:
I'm not a native speaker, but all information which I can find says it's wrong (as does codespell). See for example https://www.grammarly.com/blog/in-between-or-inbetween/.

Reply:
Do you want to keep it nevertheless?`Or replace it by in-between? Then I can update the commit.

Reply:
Ah right, should be *in between*.